uitools/toolkitwidgets/toolkitwidgets.pri

```diff
@@ -13,22 +13,26 @@
 # limitations under the License.
 include($$PWD/../common/common.pri)
 
-QT += widgets webenginewidgets svg
+android || ios {
```

Comment:
```suggestion
android|ios {
```

I am frequently confused by qmake logical operators, but I checked the doc to confirm and it's only a single `|'.
https://doc.qt.io/qt-6/qmake-language.html#:~:text=There%20is%20also%20the%20%7C%20operator

Reply:
Right. I didn't check the doc. As the code worked on my local test, I assumed it's correct.